src/cf-modals/src/cf-modals.less

```diff
@@ -0,0 +1,89 @@
+/* ==========================================================================
+   Capital Framework
+   Modal Styling
+   ========================================================================== */
+
+//
+// Theme variables
+//
+
+// .o-modal
+@modal-backdrop:   @black;
+@modal-body:       @white;
+@modal-footer:     @gray-5;
+@modal-border-top: green;
```

Comment:
missing `@` variable indicator, should be `@green`

Reply:
Fixed! Thx!